pkg/config/image.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+type ImageConfig struct {
+	// EIB Metadata
+	ApiVersion string `yaml:"apiVersion"`
```

Comment:
```suggestion
	APIVersion string `yaml:"apiVersion"`
```
Acronyms must be capitalised. Linters will complain otherwise.